app/example_user.rb

```diff
@@ -0,0 +1,26 @@
+class User
```

Comment:
Just checking if this file should be at `app/example_user.rb`? This looks like a class representing a domain model, and these live under `app/models` in Rails

Reply:
or this was an example model, was meant to remove it.